arb_os/evmOps.mini

```diff
@@ -350,11 +338,15 @@ public impure func evmOp_extcodehash(addrAsUint: uint) -> bytes32 {
         if (account_isEmpty(acct)) {
             return bytes32(0);
         }
-        if let Some(code) = account_getCode(acct) {
-            return keccak256(code, 0, bytearray_size(code));
+        return xif (evmCallFrame_constructorIsOnStackForAddress(topFrame, addr)) {
+            bytes32(0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470)  // hash of empty code
```

Comment:
Should this return the empty code hash or `0`? EIP-1052 isn't clear on this. It doesn't seem like `account_isEmpty` will ever trigger in this case, assuming `contractInfo` is set at this point.

Reply:
Agree that the EIP is unclear.  It seems safer to leave this as is.